From bf36f6712f7685e68fc46d28edfa314e4e044c99 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Thu, 6 Dec 2018 15:34:27 -0800
Subject: [PATCH] MANTA-4030 many images using node 0.10 or 0.12 broken by
 transitive lru-cache@4.1.4 dep (pinning lru-cache version is no longer
 necessary after 4.1.5 release)

---
 package.json | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/package.json b/package.json
index b51cc45..89d9a9b 100644
--- a/package.json
+++ b/package.json
@@ -1,10 +1,9 @@
 {
     "name": "firewaller",
     "description": "Triton Data Center firewalling agent",
-    "version": "1.5.1",
+    "version": "1.5.2",
     "author": "Joyent (joyent.com)",
     "private": true,
-    "// lru-cache": "attempting to pin version while satisfying transitive deps, because lru-cache@4.1.4 includes yallist@3 which breaks node --abort-on-uncaught-exception with node <v4",
     "dependencies": {
         "assert-plus": "1.0.0",
         "backoff": "2.3.0",
@@ -13,7 +12,6 @@
         "fast-stream": "git+https://github.com/joyent/sdc-fast-stream.git#893d59d",
         "jsprim": "0.5.0",
         "lockfd": "~1.2.1",
-        "lru-cache": "4.1.3",
         "restify": "2.6.1",
         "sdc-clients": "12.1.1",
         "uuid": "3.0.1",
-- 
2.21.0

